sdk/samples/iot/pnp/pnp_mqtt_message.h

```diff
@@ -9,19 +9,36 @@
 #include <azure/core/az_result.h>
 #include <azure/core/az_span.h>
 
+#define PNP_MQTT_TIMEOUT_RECEIVE_MAX_MESSAGE_COUNT 3
+#define PNP_MQTT_TIMEOUT_RECEIVE_MS (8 * 1000)
+#define PNP_MQTT_TIMEOUT_DISCONNECT_MS (10 * 1000)
+
 typedef struct
 {
   char* topic;
-  size_t topic_length;
   size_t* out_topic_length;
-  az_span payload_span;
-  az_span out_payload_span;
+  size_t topic_length;
+  az_span out_payload;
+  az_span payload;
 } pnp_mqtt_message;
 
-void pnp_mqtt_message_init(pnp_mqtt_message* mqtt_message);
+/*
```

Comment:
Thanks for doing all this work but a couple comments.

1. Note that this won't be rendered in doxygen at all since it isn't part of public API. Still useful to have since it shows what the function does.
2. IF it were intended to be processed by doxygen, note that the opening comment must be of the form 
```c
/**
```
instead of just one star. Doxygen processes double star and not single.

Reply:
So no changes needed

Reply:
oh. lol.  thanks!